e2e/capacity/transactions.test.ts

```diff
@@ -71,7 +71,7 @@ describe('Capacity Transactions', function () {
         const capacityKeys = createKeys('CapacityKeysNSF');
         const capacityProvider = await createMsaAndProvider(fundingSource, capacityKeys, 'CapProvNSF', FUNDS_AMOUNT);
         // this will first fund 'capacityKeys' before staking
-        await assert.doesNotReject(stakeToProvider(fundingSource, capacityKeys, capacityProvider, amountStaked));
+        await assert.doesNotReject(stakeToProvider(fundingSource, fundingSource, capacityProvider, amountStaked));
```

Comment:
Staking to the provider from the provider keys requires first transferring tokens to that provider. However, for most tests there is no reason to have the provider be the account doing the staking. Staking from the funding source saves that transfer transaction.